src/xdp-utils.c

```diff
@@ -978,7 +978,7 @@ xdp_connection_get_pidfd (GDBusConnection  *connection,
 }
 
 static XdpAppInfo *
-lookup_cached_app_info_by_sender (const char *sender)
+cache_lookup_app_info_by_sender (const char *sender)
```

Comment:
I don't see a need to rename this function, actually

Reply:
I did it so that  `cache_lookup_app_info_by_sender` and `cache_insert_app_info` both have the same prefix. In general we call things "object_verb" so this seems fitting, no?

Reply:
I'd say that applies if the function was acting on an object, e.g. ` cache_lookup_app_info_by_sender (cache, sender);`

Reply:
the object is "global" but if you insist I can change it back

Reply:
No hard opinions on my side